tests/unit/test_endpoint_consumer.py

```diff
@@ -179,7 +184,12 @@ def setup_charm_relations(self, multi=False):
             )
             self.harness.add_relation_unit(rel_id, "other-consumer/0")
             self.harness.update_relation_data(
-                rel_id, "other-consumer/0", {"prometheus_scrape_host": "2.2.2.2"}
+                rel_id,
+                "other-consumer/0",
+                {
+                    "prometheus_scrape_host_address": "2.2.2.2",
+                    "prometheus_scrape_host_name": "other-consumer/0",
```

Comment:
Bad name, it is a `unit_name`, not a `host_name`

Reply:
Ok will change.